### ISSUE TITLE
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.

### ISSUE THREAD

**Issue by petrussola** [PowerShell/PSReadLine#1782]
Just the usual on my VSCode 1.48.2 on my Windows Machine:

`git add .`
`git commit -m "blaba"`

When typing the first `"` the error below triggers and I get message `Oops, something went wrong.  Please report this bug with the details below. Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new  `. Find details of the error below:

```Last 50 Keys:
 g i t Space a d d Space . Enter
 g i t Space c o m m i t Space - m Space " g i t Space a d d Space . Enter
 g i t Space c o m m i t Space - m Space "

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)       
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)```

![image](https://user-images.githubusercontent.com/7245156/91621637-b2571980-e98b-11ea-9ca5-68f0b33b58da.png)

As a workaround single quotes works.

**Comment by vexx32** [comment]
See #1306 🙂

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.4 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
